### PR TITLE
Offhand Rewrite/Added Features

### PR DESCRIPTION
**Pull request by CrytoPal**
## Type of change

- [x] New feature

## Description

Rewrote Offhand which adds Right Click Gapple, Potion Offhand, and Auto Totem in one module

# How Has This Been Tested?

Tested it in AC testing servers + CC (Crystalpvp.cc)

# Videos or screenshots of the changes if applicable.

You can try out the module yourself (I was too lazy to record)

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have added comments to my code in more complex areas.
- [x] I have tested the code in both development and production environments.

### CODE REVIEW

File: src/main/java/meteordevelopment/meteorclient/systems/modules/combat/Offhand.java [hunk 1]
```diff
@@ -147,20 +204,22 @@ private boolean usableItem() {
...
     public String getInfoString() {
-        return item.get().name();
+        return preferredItem.get().name();
     }
 
     public enum Item {
-        EGap(Items.ENCHANTED_GOLDEN_APPLE),
-        Gap(Items.GOLDEN_APPLE),
+        eGap(Items.ENCHANTED_GOLDEN_APPLE),
+        gap(Items.GOLDEN_APPLE),
```
Comment (GreemDev) [inline src/main/java/meteordevelopment/meteorclient/systems/modules/combat/Offhand.java:212]: why

Comment (CrytoPal) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/combat/Offhand.java:212]: camelCasing

Comment (Wide-Cat) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/combat/Offhand.java:212]: You shouldn't really use camel case for an enum

Comment (CrytoPal) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/combat/Offhand.java]: Fixed, Ty for the feedback

**Comment by CrytoPal** [comment]
Closing to make this more organized.

**Pull request closed without merging**